doc/nrf/links.txt

```diff
@@ -382,6 +382,8 @@
 
 .. _`ZBOSS Open Initiative (ZOI)`: https://dsr-zoi.com/
 
+.. _`NCP Host for Zigbee`: https://developer.nordicsemi.com/Zigbee/ncp_sdk_for_host/ncp_sdk_v0.9.2.zip
```

Comment:
Consider a link just to ```https://developer.nordicsemi.com/Zigbee/ncp_sdk_for_host/``` rather than to a particular version.

Reply:
Yes, I've considered it, and this is the most streamlined solution now. The question is: how is the host going to be updated? New versions will replace old ones and will be backwards compatible with previous releases of NCS, or there will potentially be different versions for each NCS release? From what I was told during some review meetings, there will be new versions, and if we provide link to the directory, we will have to inform users about the compatible version somewhere.

Reply:
ok